src/components/ListView.js

```diff
@@ -0,0 +1,23 @@
+import { h } from 'preact'
+
+/**
+ * The containerRef is suggested to be used together with the hooks useNavigation
+ * without the containerRef, the view won't scroll to the selected row
+ */
+export const ListView = ({ items, containerRef }) => {
+  if (!items.length) return
+  return <div class='results' ref={containerRef}>
+    {
+      items.map(item => (
+        <div class='result' data-selectable data-title={item.title} data-selected-key={item.title} key={item.title}>
```

Comment:
The assumed structure of item correspond to a search results item. Do you plan to factor out the details of `item` out of this component later or do you anticipate that this structure will actually work for all ListView usages?

Reply:
currently, what I foresee the common list view in KaiOS, items are all selectable, with the title, optional description, optional image.

I think we can do more changes when we use it on the other pages with different requirements.